buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -89,6 +91,15 @@ abstract class AndroidTarget extends JavaLibTarget {
         // Check if kotlin
         isKotlin = project.plugins.hasPlugin(KotlinAndroidPluginWrapper.class)
         hasKotlinAndroidExtensions = project.plugins.hasPlugin(KOTLIN_ANDROID_EXTENSIONS_MODULE)
+        try {
```

Comment:
You do not need to try if `hasKotlinAndroidExtensions` is true as the extension is guaranteed to be available

Reply:
👍